en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-configuration.md

```diff
@@ -0,0 +1,22 @@
+# ISO8583 Connector Configuration
```

Comment:
Rename to "Setting up ISO8583" or something similar.

Reply:
Done.